src/Altinn.App.Core/Internal/Language/ApplicationLanguage.cs

```diff
@@ -34,17 +34,16 @@ public async Task<List<Models.ApplicationLanguage>> GetApplicationLanguages()
             var directoryInfo = new DirectoryInfo(pathTextsResourceFolder);
             var textResourceFilesInDirectory = directoryInfo.GetFiles();
             var applicationLanguages = new List<Models.ApplicationLanguage>();
+            JsonSerializerOptions options = new() { PropertyNamingPolicy = JsonNamingPolicy.CamelCase };
```

Comment:
Nice catch!

Reply:
This is at least a small performance boost, as `System.Text.Json` caches some of it's reflection results on the `JsonSerializerOptions` instance.